bin/index.js

```diff
@@ -41,7 +42,7 @@ if (argv.s) {
 } else {
   if (!argv.i || !argv.o) argvError();
 
-  renderAndBuild(argv.i, argv.c, argv.o, argv.l, argv.S)
+  renderAndBuild(argv.i, argv.c, argv.f, argv.o, argv.l, argv.S)
```

Comment:
This option applies only to the build mode (renderAndBuild) and does not propagate to the dev mode (renderAndServe), because in server mode it would require to intercept a request for `index.html` and to modify it in a fly.

I thought this is not so necessary to do for a such minor option